packages/core/src/base/typography/link.scss

```diff
@@ -27,7 +27,7 @@
 }
 
 // Change color of links that are descendants of .ds-base--inverse
-.ds-base--inverse a {
+.ds-base--inverse a, .ds-c-link--inverse {
```

Comment:
The default A tag is still styled by default and also is still styled as a child of `.ds-base--inverse`